examples/mux-server/go.mod

```diff
@@ -23,11 +23,10 @@ require (
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.46.0 // indirect
 	go.opentelemetry.io/contrib/propagators/b3 v1.20.0 // indirect
 	go.opentelemetry.io/otel v1.20.0 // indirect
-	go.opentelemetry.io/otel/exporters/otlp/otlpmetric v0.42.0 // indirect
-	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.42.0 // indirect
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.43.0 // indirect
```

Comment:
i think you need to rerun go mod tidy in these sub folders? eg. go.opentelemetry.io/contrib/propagators/b3 v1.20.0 should be at 1.21.0.

Reply:
I ran make tidy, let me try running go mod tidy on each of them one by one

Reply:
oh i did not realize you can not updated the go.opentelemetry.io/contrib/propagators/b3 to v1.21.0